tpot/tpot.py

```diff
@@ -247,6 +246,10 @@ def fit(self, features, classes):
         None
 
         """
+        
+        if self.expert_source != None:
+            self._ekf(features, self.ekf_index, self.k_best)        
+        
```

Comment:
We should do some checks on the `expert_source` values at `fit` time. In particular:

1) Make sure all EK sources are of the same length as the input features

2) Make sure all EK sources with only the values `0` and `1` are encoded as `bool`

3) Make sure there's no missing values in the EK sources